### ISSUE TITLE
Lines with trailing comments are being lost

### ISSUE THREAD

**Issue by JoBoDo** [JankGaming/vdfjs#5]
Keys such as "ModelScale" and "Ability1" are not found in json,
however "ClassTier"  and "AbilityLayout" are

Example of VDF:

```
"npc_dota_hero_arcanist"
    {

        "override_hero"             "npc_dota_hero_crystal_maiden"          // Hero to override
        "ModelScale"                ".75"
        "ClassTier"                 "3"
        "DrawParticlesWhileHidden" "1"

        "AbilityLayout"             "5"
        "Ability1"                  "customability_arcanebarrage"           // Ability 1
        "Ability2"                  "customability_essenceredirect"     // Ability 2
        "Ability3"                  "customability_arcanepulse"         // Ability 3
        "Ability4"                  "customability_essenceshift"            // Ability 4
        "Ability5"                  "customability_spellbook"               // Ability 5    
    }
```

Example of JSON:

```
{
  "npc_dota_hero_arcanist": {
    "ModelScale": ".75",
    "ClassTier": "3",
    "DrawParticlesWhileHidden": "1",
    "AbilityLayout": "5"
  }
}
```

**Comment by twisterghost** [comment]
👍 will look into it

**Comment by twisterghost** [comment]
Fixed in [2.0.1](https://github.com/JankGaming/vdfjs/releases/tag/v2.0.1)
